app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app import errors
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
```

Comment:
The method returns `str` type, raising an error does not imply that functions return type will be `None`